### PR TITLE
Feature/add totorial page

### PR DESCRIPTION
**Pull request by AmirHalabi**
add new tutorial client side in the mean time

### CODE REVIEW

File: src/app/core/layouts/navigation-bar/navigation-bar.component.ts [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
 })
 export class NavigationBarComponent implements OnInit {
+  constructor(public dialog: MatDialog) {}
 
-  constructor() { }
-
-  ngOnInit() {
+  ngOnInit() {}
+  openDialog() {
+    console.log('open sesemy');
```
Comment (OfirCohen28101999) [inline src/app/core/layouts/navigation-bar/navigation-bar.component.ts]: should we leave console logs in the code for now?

File: src/app/features/new-tutorial-dialog/new-tutorial-dialog.component.html [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+          >Text</mat-radio-button
+        >
+      </mat-radio-group>
+    </div>
+    <div *ngIf="submitClicked">
+      <div *ngIf="f.videoOrText.errors?.required" class="error-message">
+        required field.
+      </div>
+    </div>
+    <div class="raio-button-area">
```
Comment (OfirCohen28101999) [inline src/app/features/new-tutorial-dialog/new-tutorial-dialog.component.html]: radio*

File: src/app/core/layouts/navigation-bar/navigation-bar.component.ts (hunk 1 above)
Comment (AmirHalabi) [reply src/app/core/layouts/navigation-bar/navigation-bar.component.ts]: no i will fix it, 
we should only do errors in the mean time

File: src/app/features/new-tutorial-dialog/new-tutorial-dialog.component.html (hunk 2 above)
Comment (AmirHalabi) [reply src/app/features/new-tutorial-dialog/new-tutorial-dialog.component.html]: fixed :)

**Pull request merged**